# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mythtv-plugins/fix-mythweb-in-0.20.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mythplugins-0.20/mythweb/includes/recording_schedules.php	2006-09-10 17:08:56.000000000 -0400
+++ mythplugins-0.20-fixmythweb/mythweb/includes/recording_schedules.php	2006-09-11 14:46:10.000000000 -0400
@@ -2,9 +2,9 @@
 /**
  * The Schedule object and a couple of related subroutines.
  *
- * @url         $URL: svn+ssh://ijr@cvs.mythtv.org/var/lib/svn/trunk/mythplugins/mythweb/includes/recording_schedules.php $
- * @date        $Date: 2006-09-10 17:08:56 -0400 (Sun, 10 Sep 2006) $
- * @version     $Revision: 11113 $
+ * @url         $URL: svn+ssh://ijr@cvs.mythtv.org/var/lib/svn/branches/release-0-20-fixes/mythplugins/mythweb/includes/recording_schedules.php $
+ * @date        $Date: 2006-09-11 14:46:10 -0400 (Mon, 11 Sep 2006) $
+ * @version     $Revision: 11150 $
  * @author      $Author: xris $
  * @license     GPL
  *
@@ -255,7 +255,7 @@
                                        VALUES (?,?,?,
                                                FROM_UNIXTIME(?),FROM_UNIXTIME(?),FROM_UNIXTIME(?),FROM_UNIXTIME(?),
                                                ?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)',
-                         $this->recordid,
+                         _or($this->recordid, 0),
                          $this->type,
                          $this->chanid,
                          $this->starttime,
@@ -270,9 +270,9 @@
                          $this->recpriority,
                          $this->category,
                          $this->maxnewest,
-                         $this->inactive,
+                         _or($this->inactive, 0),
                          $this->maxepisodes,
-                         $this->autoexpire,
+                         _or($this->autoexpire, 0),
                          $this->startoffset,
                          $this->endoffset,
                          $this->recgroup,
@@ -281,18 +281,18 @@
                          $this->station,  // callsign!
                          $this->seriesid,
                          $this->programid,
-                         $this->autocommflag,
+                         _or($this->autocommflag, 0),
                          $this->findday,
                          $this->findtime,
                          $this->findid,
-                         $this->autotranscode,
+                         _or($this->autotranscode, 0),
                          $this->transcoder,
-                         $this->parentid,
+                         _or($this->parentid, 0),
                          $this->tsdefault,
-                         $this->autouserjob1,
-                         $this->autouserjob2,
-                         $this->autouserjob3,
-                         $this->autouserjob4
+                         _or($this->autouserjob1, 0),
+                         _or($this->autouserjob2, 0),
+                         _or($this->autouserjob3, 0),
+                         _or($this->autouserjob4, 0)
                         );
     // Get the id that was returned
         $recordid = $sh->insert_id();
